app/models/concerns/validating_enum.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+# Ref: https://medium.com/nerd-for-tech/using-activerecord-enum-in-rails-35edc2e9070f
+module ValidatingEnum
+  # Macro that wraps the standard Rails `enum` with additional type behavior to support
+  # intutive validation of enum selections
+  #
+  # Be sure to add `extend ValidatingEnum` to each model using this
```

Comment:
If we go with the Railtie approach to auto-extend `ValidatingEnum` does that mean we can delete this line?